meta/main.yml

```diff
@@ -2,11 +2,11 @@
 galaxy_info:
   role_name: "jenkins"
   author: "Lean Delivery team <team@lean-delivery.com>"
-  description: "Lean Delivery Jenkins installation and configuration"
+  description: "Jenkins installation and configuration"
   company: "Epam Systems"
   license: "Apache"
   min_ansible_version: "2.5"
-  issue_tracker_url: "https://github.com/lean-delivery/ansible-role-jenkins/issues"
+  issue_tracker_url: "https://github.com/lean-delivery/ansible-role-jenkins2/issues"
```

Comment:
link `https://github.com/lean-delivery/ansible-role-jenkins/issues2` returns 404
Is it ok?

Reply:
fixed